spec/services/planned_disbursement_history_spec.rb

```diff
@@ -122,6 +122,22 @@ def history_entries
 
         expect(history_entries).to eq([])
       end
+
+      it "creates a public activity record for each deleted entry" do
+        PublicActivity.with_tracking do
+          history.set_value(10)
+          history.set_value(5)
+
+          old_entries = history.all_entries
+
+          history.clear
+
+          old_entries.each do |entry|
+            auditable_event = PublicActivity::Activity.find_by(trackable_id: entry.id, parameters: {associated_activity_id: activity.id})
+            expect(auditable_event).to_not be_nil
```

Comment:
Since `find_by` will blow up if there's no matching entry, I'm wondering if it's better to check that two `PublicActivity::Activity` records were created with the right `key` and `parameters` set?

Reply:
Ah yeah, good catch. I did think about doing that, as that's how other tests for Public Activity work.